eva/catalog/models/df_metadata.py

```diff
@@ -28,16 +28,20 @@ class DataFrameMetadata(BaseModel):
     _is_video = Column("is_video", Boolean)
     _columns = relationship(
         "DataFrameColumn",
-        back_populates="_dataset",
```

Comment:
I'm a bit confused about this design. Can we make a new model for `Dataset` and add relationships to columns and dataframes?

Reply:
Ah. I forget to push the dataset model file. Yes, there is a new model for `Dataset`